javaoop/src/main/java/bootcamp/Threads.java

```diff
@@ -2,7 +2,24 @@
 
 public class Threads{
 
-  public static void main(String[]  args){
-    System.out.println("Make this code compile and run");
+  public static void main(String[] args){
+
+      // Creates an array with 6 columns and 20 rows
+      final String[][] arrayWithoutHeader = {
+          {"Sterling B. Rosario", "Steven G. Cameron", "Amber M. Doss", "Troy D. Lee", "Brenda M. Goldman", "James M. McCracken", "Hope C. Edwards", "James M. Hicks", "Carolyn T. Muniz", "David I. Rojas", "Donald V. McKinnon", "Erik M. Robinson", "Joyce R. Murray", "William R. Levitt", "Eugene L. Johnson", "Jerry E. Dorsey", "Joseph K. Wallace", "Daisy J. Glover", "Debbie C. Allen", "David A. Bryce"},
+          {"612-388-9263", "814-577-8878", "276-859-0345", "650-687-9895", "715-469-4937", "239-732-0630", "208-549-2256", "806-794-2570", "815-772-5445", "786-236-8216", "518-701-0696", "609-546-2522", "352-408-6547", "510-927-9462", "850-941-3589", "214-887-9398", "201-326-1372", "646-277-7890", "978-254-0960", "310-312-0940"},
```

Comment:
Let's make entries of other types of information as well.
Currently I see that here is 23 entries with the same type of info: 6 columns of person info.

I want to have 23 entries with 6 fields of different type of info:
1) person has name, phone, job title...
2) car has model, fuel type, power..
3) plane has name, passenger capacity, range
....
I want to have 5-6 different data types; all of those types should have 6 columns